scan/spec/test_command_generator_spec.rb

```diff
@@ -5,6 +5,31 @@
     @project = FastlaneCore::Project.new(config)
   end
   before(:each) do
+    @valid_simulators = "== Devices ==
+-- iOS 9.3 --
+    iPhone 4s (238767C4-AF29-4485-878C-7011B98DCB87) (Shutdown)
```

Comment:
Noticed this and then noticed [this](https://github.com/fastlane/fastlane/blob/master/fastlane_core/lib/fastlane_core/device_manager.rb#L102), any reason why Fastlane isn't made Xcode 7+? Would make stuff like this a heck of a lot nicer :)